File: core/Dtos/CVReferredDto.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace core.Dtos
{
    public class CVReferredDto
    {
        public int CvRefId { get; set; }
        public string CustomerName { get; set; }
        public int OrderId { get; set; }
        public int OrderNo { get; set; }
        public DateTime OrderDate {get; set;}
        public int OrderItemId { get; set; }
        public string CategoryName { get; set; }
        public string CategoryRef { get; set; }
        public int CustomerId { get; set; }
        public int CandidateId { get; set; }
        public int ApplicationNo { get; set; }
        public string CandidateName { get; set; }
        public DateTime ReferredOn { get; set; }
        public string ReferralDecision { get; set; }
        public DateTime SelectedOn { get; set; }
        public ICollection<DeployRefDto> Deployments { get; set; }
    }

    
}

File: client2/src/app/shared/dtos/admin/cvReferredDto.ts
export interface ICVReferredDto
{
     checked: boolean;
     cvRefId: number;
     customerName: string;
     orderId: number;
     orderNo: number;
     orderDate: Date;
     orderItemId: number;
     categoryName: string;
     categoryRef: string;
     customerId: number;
     candidateId: number;
     applicationNo: number;
     candidateName: string;
     agentName: string;
     referredOn: Date;
     referralDecision: number;
     selectedOn: Date;
     deployments: IDeployDto[];
}

export class CVReferredDto implements ICVReferredDto
{
     checked = false;
     cvRefId= 0;
     customerName= '';
     orderId= 0;
     orderNo= 0;
     orderDate = new Date('1900-01-01');
     orderItemId= 0;
     categoryName= '';
     categoryRef= '';
     customerId= 0;
     candidateId= 0;
     applicationNo= 0;
     candidateName= '';
     agentName= '';
     referredOn = new Date('1900-01-01');
     referralDecision= 0;
     selectedOn = new Date('1900-01-01');
     deployments: IDeployDto[] = [];
}

export interface IDeployDto
{
     id: number;
     deployCVRefId: number;
     transactionDate: Date;
     sequence: number;
     nextSequence: number;
     nextStageDate: Date;
}